strapi/strapi
# Try to automatically clone an entity and redirect, if not, redirect to clone route

### What does it do?

* `DynamicTable` in the CM was only used in ListView. I've moved it closer to the view now
* Adds posting to clone initially on "clone click"
* Redirects you to new entity view if successful clone
* Redirects you to regular cloning route if failed clone
* Shows a basic notification for the error (BE to update this separately)

### Why is it needed?

* Cloning is a specific action, lets make it quicker!

### How to test it?

* Press clone on an entity with just a text field, it should do it automatically and redirect you to the new entity
* Press clone on an entity that has a one-to-one relation, it will fail and redirect you to the cloning view with a notification

### Related issue(s)/PR(s)

* resolves CONTENT-980
* resolves CONTENT-982
* resolves CONTENT-983
